src/moaicore/MOAIFreeTypeFont.cpp

```diff
@@ -343,10 +343,10 @@ int MOAIFreeTypeFont::_setDefaultSize(lua_State *L){
 FT_Face MOAIFreeTypeFont::AffirmFreeTypeFace () {
 	
 	if (!this->mFreeTypeFace) {
-		FT_Library library;
-		FT_Error error = FT_Init_FreeType( &library );
+		this->mLibrary = new FT_Library();
+		FT_Error error = FT_Init_FreeType( this->mLibrary );
 		CHECK_ERROR(error);
-		return this->LoadFreeTypeFace( &library );
+		return this->LoadFreeTypeFace( this->mLibrary );
```

Comment:
Why extend the lifetime of the `FT_Library` after this method call? It was getting destroyed at its end, now it's not. If it's to be able to call `FT_Done_FreeType`, why not do...
```
FT_Library library;
FT_Error error = FT_Init_FreeType( &library );
CHECK_ERROR(error);
FT_Face face = this->LoadFreeTypeFace( &library );
FT_Done_FreeType( library );
return face;
```

Does that make sense? Or we can't call `FT_Done_FreeType` here because it makes the return value useless to the caller? I'm assuming here that if the `FT_Library` was being destroyed at the end of this function it's okay to clean up here, but I don't know for sure.